Build failure: axel

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

- build *axel*

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/302515619

### Relevant log output

```console
m4/gettext.m4:57: AM_GNU_GETTEXT is expanded from...
configure.ac:239: the top level
configure.ac:240: error: possibly undefined macro: AM_GNU_GETTEXT_VERSION
If this token and others are legitimate, please use m4_pattern_allow.
See the Autoconf documentation.
autoreconf: error: /nix/store/0qmqybsb86p5g6vqxj992grj9hyh72wa-autoconf-2.72/bin/autoconf failed with exit status: 1
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.36, NixOS, 25.11 (Xantusia), 25.11.20250709.be9e214`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.3
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/misile/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/misile/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/misile/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/misile/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/3j073h93daccd3sk5nvm241sdpnd1aqz-lix-2.91.3/share`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/9g7vwfbsk7hcp12vrm8a82cnv6y9v8fb-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

I'm having the same problem and tried to solve it. Using the following on the axel derivation goes further:

```
  autoreconfFlags = ''-fvi -I ${gettext}/share/gettext/m4'';
```

To build, I've used this:

```
nix-shell -I nixpkgs=. -p axel
```

But there is still an error when building:

```
...
make  all-recursive
make[1]: Entering directory '/build/source'
Making all in po
make[2]: Entering directory '/build/source/po'
make[2]: *** No rule to make target '/config.status', needed by 'Makefile'.  Stop.
make[2]: Leaving directory '/build/source/po'
make[1]: *** [Makefile:921: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:471: all] Error 2
```

I don't have time to work on this right now, but if someone can, please share.

> I'm having the same problem and tried to solve it. Using the following on the axel derivation goes further:
> 
> ```
>   autoreconfFlags = ''-fvi -I ${gettext}/share/gettext/m4'';
> ```
> 
> To build, I've used this:
> 
> ```
> nix-shell -I nixpkgs=. -p axel
> ```
> 
> But there is still an error when building:
> 
> ```
> ...
> make  all-recursive
> make[1]: Entering directory '/build/source'
> Making all in po
> make[2]: Entering directory '/build/source/po'
> make[2]: *** No rule to make target '/config.status', needed by 'Makefile'.  Stop.
> make[2]: Leaving directory '/build/source/po'
> make[1]: *** [Makefile:921: all-recursive] Error 1
> make[1]: Leaving directory '/build/source'
> make: *** [Makefile:471: all] Error 2
> ```
> 
> I don't have time to work on this right now, but if someone can, please share.

I think we can do the same patch as https://github.com/NixOS/nixpkgs/pull/425574

--- a/pkgs/tools/networking/axel/default.nix
+++ b/pkgs/tools/networking/axel/default.nix
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-5GUna5k8GhAx1Xe8n9IvXT7IO6gksxCLh+sMANlxTBM=";
   };
 
+  postPatch = ''
+    substituteInPlace configure.ac \
+      --replace-fail "AM_GNU_GETTEXT_VERSION([0.11.1])" "AM_GNU_GETTEXT_VERSION([0.12])"
+  '';
+
   nativeBuildInputs = [
     autoreconfHook
     pkg-config

